testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_replication_eventing.py

```diff
@@ -484,3 +485,75 @@ def test_replication_delete_event(params_from_base_test_setup, num_of_docs):
     doc_ids = db.getDocIds(cbl_db)
     for sg_doc in docs_to_modify:
         assert sg_doc["id"] not in doc_ids, "channel access removal didn't purge the docs from cbl db"
+
+
+@pytest.mark.listener
+@pytest.mark.replication
+@pytest.mark.parametrize("attachment_generator, attachment_file_list", [
+    [None, None],
+    [load_from_data_dir, ["Sample_Image_30mb.jpg"]],
```

Comment:
related to same comment mentioned above. Can we have send as one file rather than having a list? Is there any advantage?

Reply:
the advantage is if in future we want to change and check with multiple files, we don't need to make any change. Program should be written from scalability perspective not just current scenario.